rep-0105.rst

```diff
@@ -51,9 +51,37 @@ therefore eliminating drift, but causing discrete jumps when new
 sensor information arrives.
 
 The ``map`` frame is useful as a long-term global reference, but
-discrete jumps make it a poor reference frame for local sensing and
+discrete jumps in position estimators make it a poor reference frame for local sensing and
 acting.
 
+**Map Conventions**
+
+Map coordinate frames can either be referenced globally or to an application specific position.
+
+When defining coordinate frames with respect to a global reference like the earth.
```

Comment:
Perhaps the period should be colon, and the next two sentences should be items in a bulleted list?